api/data.go

```diff
@@ -67,3 +67,24 @@ func (api *API) AddDataSets(ctx context.Context) http.HandlerFunc {
 		}
 	}
 }
+
+// GetCacheTime retrieves cache time data for a given ID and writes it to the HTTP response.
+func (api *API) GetCacheTime(ctx context.Context, w http.ResponseWriter, req *http.Request) {
+	log.Info(ctx, "Calling get cache time handler")
+
+	vars := mux.Vars(req)
+	id := vars["id"]
+
+	cacheTime, err := api.dataStore.GetCacheTime(ctx, id)
+	if err != nil {
+		log.Error(ctx, "Error retrieving cache time from datastore", err)
```

Comment:
logs start lowercase. check all logs here in your PR. More than one change is required here. 

ONS [logging standards](https://github.com/ONSdigital/dp-standards/blob/main/LOGGING_STANDARDS.md#effective-event-types)